CMake/SlicerCPack.cmake

```diff
@@ -338,8 +338,8 @@ slicer_cpack_set("CPACK_RESOURCE_FILE_LICENSE")
 get_property(${app_name}_CPACK_PACKAGE_DESCRIPTION_SUMMARY GLOBAL PROPERTY ${app_name}_DESCRIPTION_SUMMARY)
 slicer_cpack_set("CPACK_PACKAGE_DESCRIPTION_SUMMARY")
 
-get_property(${app_name}_CPACK_PACKAGE_ICON GLOBAL PROPERTY ${app_name}_APPLE_ICON_FILE)
 if(APPLE)
+  get_property(${app_name}_CPACK_PACKAGE_ICON GLOBAL PROPERTY ${app_name}_APPLE_ICON_FILE)
```

Comment:
Since the variable `APPLE_ICON_FILE` is expected by `slicerMacroBuildApplication`, I suggest to keep this code as is.

https://github.com/Slicer/Slicer/blob/224995ba039f5bc540197a4aa94e393164106d0a/Applications/SlicerApp/CMakeLists.txt#L66-L78

Reply:
Ok I've reverted the change.